idl/idl-models/src/org/jetbrains/dukat/idlModels/convertToModel.kt

```diff
@@ -29,12 +43,26 @@ fun IDLDeclaration.convertToModel(): TopLevelModel? {
                 annotations = mutableListOf(),
                 external = false
         )
+        is IDLAttributeDeclaration -> PropertyModel(
+                name = IdentifierEntity(name),
+                type = TypeValueModel(
+                        value = IdentifierEntity(convertPrimitiveType(type.name)),
+                        params = listOf(),
+                        metaDescription = null
+                ),
+                typeParameters = listOf(),
+                static = false,
+                override = false,
+                getter = false,
+                setter = false,
+                open = false
+        )
         else -> raiseConcern("unprocessed MemberNode: ${this}") { null }
     }
 }
 
-fun IDLFileDeclaration.process(): SourceSetModel {
-    val modelDeclarations = declarations.mapNotNull { d -> d.convertToModel() }
+fun IDLFileDeclaration.process() : SourceSetModel {
+    val modelDeclarations = declarations.mapNotNull { d -> d.convertToModel() }.map { d -> d as TopLevelModel }
```

Comment:
please, choose one of two:

- leave it as `it` param
- use `declaration` as a param name

nobody benefits of having it as d.
This is is general.
In this particular case - why this conversion happens at all PropertyModel is never a TopLevelModel.